lib/address_geocoder.rb

```diff
@@ -30,46 +30,32 @@ def initialize(opt = {})
   end
 
   def valid_address?
-    # 1. If address values have changed call google
-    call_google if values_changed?
+    # 1. If address values have changed call third_party_api
+    call_map_apis if values_changed?
     # 2. Return T/F depending on success of call and certainty of success
     @response.success? && @response.result['certainty']
   end
 
   def suggested_addresses
-    # 1. If address values have changed call google
-    call_google if values_changed?
+    # 1. If address values have changed call third_party_api
+    call_map_apis if values_changed?
     # 2. If response failed return false
     return false unless @response.success?
     # 3. Initialize refined_address
     country_wo_postal = match_country.reject { |k| k == :postal_code }
     refined_address = { country: country_wo_postal, city: nil, state: nil, postal_code: nil, street: nil }
-    # 4. Pass refined address and google response to parser
+    # 4. Pass refined address and third_party_api response to parser
     parser = Parse.new(@response.result['results'][0]['address_components'], refined_address)
```

Comment:
Unless we standardize results completely we're going to have to make a new parser method